README.md

```diff
@@ -86,11 +86,11 @@ React Hooks library for data fetching from Vercel
 
 [Go To Documentation](https://swr.vercel.app/)
 
-### **React Redux**
+### **Unstated-Next**
```

Comment:
Do we need to have this bit for every plugin? I think it would be better if we remove it to make our README a bit cleaner. Let me know if you want to do this in a separate PR though.

Reply:
This was just to remove every instance of Redux. The task for updating the README is still not yet to be done yet.

Reply:
Yes, it should be done in a separate PR. I believe I made it a task on the dev board but it's not assigned to anyone currently.